### PR TITLE
Unify v020 readmes

### PR DESCRIPTION
**Pull request by Fzilan**

### CODE REVIEW

File: examples/animatediff/README.md [hunk 1]
```diff
@@ -252,41 +252,41 @@ Here are some generation results after lora fine-tuning on 512x512 resolution an
...
 ### Inference
 
 - mindspore 2.3.1
 
-|   model name   | mindspore | scheduler | steps | resolution | frame | speed (step/s) | time(s/video) |
-|:--------------:|:---------:|:---------:|:-----:|:----------:|:-----:|:--------------:|:-------------:|
-| AnimateDiff v2 |   2.3.1   |   DDIM    |  30   |  512x512   |  16   |      0.6       |      18       |
+|   model name   |cards| resolution  | scheduler | steps |     s/step     |    s/video    |
+|:--------------:|:--:|:----------: |:---------:|:-----:|:--------------:|:-------------:|
+| AnimateDiff v2 |1|  512x512x16 |   DDIM    |  30   |      0.60      |      18.00    |
```
Comment (SamitHuang) [inline examples/animatediff/README.md:263]: 0.6 step/s  = 1/0.6 s/step

File: examples/animatediff/README.md [hunk 2]
```diff
@@ -252,41 +252,41 @@ Here are some generation results after lora fine-tuning on 512x512 resolution an
 
 ## Performance (AnimateDiff v2)
 
-Experiments are tested on ascend 910* graph mode with a single card.
+Experiments are tested on ascend 910* graph mode.
```
Comment (SamitHuang) [inline examples/animatediff/README.md:255]: mention ms2.3.1?

File: examples/diffusers/controlnet/README.md [hunk 3]
```diff
@@ -53,11 +53,7 @@ python train_controlnet.py \
  --train_batch_size=4
 ```
 
-This default configuration requires ~38GB VRAM.
-
-By default, the training script logs outputs to tensorboard.
-
-Gradient accumulation with a smaller batch size can be used to reduce training requirements to ~20 GB VRAM.
+Gradient accumulation with a smaller batch size can be used to reduce training requirements,
```
Comment (SamitHuang) [inline examples/diffusers/controlnet/README.md:56]: memory requirement

File: examples/diffusers/dreambooth/README.md [hunk 4]
```diff
@@ -309,7 +309,7 @@ Additionally, a few alternative cli flags are needed for IF.
 
 `--resolution=64`: IF is a pixel space diffusion model. In order to operate on un-compressed pixels, the input images are of a much smaller resolution.
 
-`--pre_compute_text_embeddings`: IF uses [T5](https://huggingface.co/docs/transformers/model_doc/t5) for its text encoder. In order to save GPU memory, we pre compute all text embeddings and then de-allocate
+`--pre_compute_text_embeddings`: IF uses [T5](https://huggingface.co/docs/transformers/model_doc/t5) for its text encoder. In order to save memory, we pre compute all text embeddings and then de-allocate
```
Comment (SamitHuang) [inline examples/diffusers/dreambooth/README.md:312]: pre-compute

File: examples/dit/README.md [hunk 5]
```diff
@@ -81,9 +81,9 @@ ddim_sampling: True
 
 Experiments are tested on ascend 910* with mindspore 2.3.1 graph mode:
 
-| model name | cards | image size | method | steps | jit level | ckpt loading time | graph compile | sample time |
-| :--------: | :---: | :--------: | :----: | :---: | :-------: | :---------------: | :-----------: | :---------: |
-|    dit     |   1   |  256x256   |  ddpm  |  250  |    O2     |      16.41s       |    82.83s     |   58.45s    |
+| model name | cards | resolution | scheduler | steps | jit level | graph compile | s/img |
+| :--------: | :---: | :--------: | :----: | :---: | :-------: | :-----------: | :---------: |
+|    dit     |   1   |  256x256   |  ddpm  |  250  |    O2     |    82.83s     |   58.45    |
```
Comment (SamitHuang) [inline examples/dit/README.md:86]: check s/img for how many steps?

File: examples/animatediff/README.md (hunk 1 above)
Comment (Fzilan) [reply examples/animatediff/README.md:263]: 重新测过，之前单位写错了。

File: examples/animatediff/README.md (hunk 2 above)
Comment (Fzilan) [reply examples/animatediff/README.md:255]: 放了两个版本的performance，下面两个表分别在表格上方提了。

File: examples/diffusers/controlnet/README.md (hunk 3 above)
Comment (Fzilan) [reply examples/diffusers/controlnet/README.md:56]: to be tested.

File: examples/dit/README.md (hunk 5 above)
Comment (Fzilan) [reply examples/dit/README.md:86]: 表格写了 steps - 250

File: examples/diffusers/dreambooth/README.md (hunk 4 above)
Comment (Fzilan) [reply examples/diffusers/dreambooth/README.md]: fixed

File: examples/animatediff/README.md (hunk 1 above)
Comment (SamitHuang) [inline examples/animatediff/README.md:263]: 这里，0.6step/s，对应 1.67s/step，总的采样时间对应 1.67*30=50.1s

Comment (SamitHuang) [reply examples/animatediff/README.md:263]: ok

**Review by SamitHuang — APPROVED** [review]

**Review by CaitinZhao — APPROVED** [review]
